imports/plugins/core/files/server/no-meteor/setUpFileCollections.js

```diff
@@ -81,7 +81,7 @@ export default function setUpFileCollections({
         fileRecord.extension(format, { store: name });
 
         // resizing image, adding mod, setting output format
-        return sharp().resize(size, size)[mod]().toFormat(format);
+        sharp().resize(size, size)[mod]().toFormat(format);
```

Comment:
This needs to return a ReadWrite stream or transformation won't work. See https://github.com/reactioncommerce/reaction-file-collections#create-stores

Looking at [the package code](https://github.com/reactioncommerce/reaction-file-collections/blob/83078b95db6bfb1c36cd2d4bb80ba8ed4d45fd35/packages/file-collections-sa-base/src/StorageAdapter.js#L142-L147), it only does a truthy check, so changing the first line here to `return null` is fine.